lib/api/html/functions.php

```diff
@@ -137,15 +137,15 @@ function beans_open_markup( $id, $tag, $attributes = array() ) {
 
 	// Skip the opening tag when it's empty.
```

Comment:
The comment does not explain what is happening below.

Reply:
Sure it does.  We _skip_ building the opening tag when the tag is empty.

Reply:
I changed this one to `// Build the opening tag when tag is available.` and applied it to the closing tag as well.